File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Sys_GroupMenusService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Sys_GroupMenusService : BaseService<Sys_GroupMenus, ISys_GroupMenusRepository>
    , ISys_GroupMenusService
    {
    private readonly ISys_GroupMenusRepository _Sys_GroupMenusRepository;
    public Sys_GroupMenusService(ISys_GroupMenusRepository Sys_GroupMenusRepository) : base(Sys_GroupMenusRepository)
    {
    _Sys_GroupMenusRepository = Sys_GroupMenusRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Sys_GroupMenus.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_GroupMenus } from './Sys_GroupMenus.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_GroupMenusService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Group_Id', 'Menu_Id'];

    List: Sys_GroupMenus[] | undefined;
    ListFilter: Sys_GroupMenus[] | undefined;
    FormData!: Sys_GroupMenus;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_GroupMenus";
    }
}

